### DISCUSSION TITLE
Moved plots from one HDD to another, now they have become invalid. Please help.

### DISCUSSION THREAD

**Discussion by LiuKang6** [Chia-Network/chia-blockchain#6283] [Support]
Hey everyone,

I moved some plots from one HDD to another one to create some space. Now they have become invalid plots. How can the System actually recognize that those plots are not anymore in HDD E: but in F:? As you can see on the picture it is stated as multiple copies even though it is not the case.

Thanks for your help guys, i appreciate it god bless you all.

![xch problem](https://user-images.githubusercontent.com/85136813/120246203-d673be00-c26f-11eb-8a30-ff4cf625a77b.jpg)

**Comment by BrandtH22** [comment]
@LiuKang6 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Note - you may want to remove the plot directories and readd them to force the client to rescan the directories themselves 👍 

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
